pro_tes/middleware/middleware.py

```diff
@@ -13,14 +14,22 @@ class AbstractMiddleware(metaclass=abc.ABCMeta):
 
     @abc.abstractmethod
     def modify_request(self, request):
-        """Modify the request before it is sent to the TES instance."""
+        """Abstract method to modify the incoming request.
```

Comment:
Docstring summaries should start with a verb, in imperative form. So better to use:

```
"""Modify the incoming task request.

Abstract method.

Args:
    ...
"""
```

Reply:
Done

Reply:
Not really. It doesn't start with a verb in imperative form. I mean it does, technically, because "abstract" can also be a verb, but here it is meant as an adjective. I have already written how to write it instead. Just use that.

Reply:
ooh Sorry, for that now fixed